cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -267,6 +267,12 @@ def test_provision_from_template_using_rest(appliance, request, provider, vm_nam
         initialEstimate: None
```

Comment:
would be nice to add some metadata here :)

Reply:
I have another PR for all the Polarion refactoring. I could use your inputs there as well.
PR #8268